package.json

```diff
@@ -56,6 +56,9 @@
     "heroku-spaces": "2.9.5",
     "semver": "5.3.0"
   },
+  "devDependencies": {
+    "heroku-plugin-readme-generator": "git+ssh://git@github.com/heroku/heroku-cli-dev-center-generator.git"
```

Comment:
CI is failing on appveyor because it doesn't seem to work with a git reference. I published this to npm though, so change this reference to use the package and it should be resolved